src/main/java/io/cdap/plugin/format/error/ErrorSchema.java

```diff
@@ -0,0 +1,35 @@
+package io.cdap.plugin.format.error;
+
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.cdap.api.data.schema.Schema;
+
+import javax.annotation.Nullable;
+
+/**
+ * TODO:add
+ */
+public class ErrorSchema {
+  public static final String SCHEMA_NAME = "--io.cdap.cdap.api.data.schema.Schema.ErrorSchema";
```

Comment:
I'm using this as the schema name as a way to prevent conflicts between any possible SQL table name.